apps/ui/src/views/Proposal.vue

```diff
@@ -157,11 +159,14 @@ watchEffect(() => {
 </script>
 
 <template>
-  <div class="flex items-stretch md:flex-row flex-col w-full md:h-full !pb-0">
+  <div class="flex items-stretch md:flex-row flex-col w-full h-full">
     <UiLoading v-if="isPending" class="ml-4 mt-3" />
     <template v-else-if="proposal">
       <div
-        :class="['flex-1 grow min-w-0', { '!pb-0': withoutBottomPadding }]"
+        :class="[
```

Comment:
There's extra scroll in the votes page, see http://localhost:8080/#/s:balancer.eth/proposal/0x028874ebece2c48caf0b19d4c147a4946e71b16c2518cb5a6148798455832625/votes